Source/Scene/Model.js

```diff
@@ -2943,9 +2946,6 @@ define([
                                 BoundingSphere.clone(command.boundingVolume, pickCommand.boundingVolume);
                             }
                         }
-                    } else {
```

Comment:
I deleted it because I ended up moving that code to line 2927, which all node types will call.